konig-omcs-model/src/main/java/io/konig/omcs/datasource/OracleTableReference.java

```diff
@@ -33,6 +33,9 @@ public OracleTableReference() {
 	}
 	
 	public String getOracleHost() {
+		if(System.getProperty(oracleHost) != null) {
+			oracleHost = System.getProperty(oracleHost);
+		}
 		return oracleHost;
```

Comment:
In this block of code you are using `oracleHost` as (1) the name of a property and (2) the value of the property.  This is a bad practice.  Please revise the solution so that there are two fields: one that holds the name of the property and one that holds he value of the property.